src/arrays/programs_involving_data_sequences/hard/q9/MainTest.java

```diff
@@ -0,0 +1,79 @@
+package arrays.programs_involving_data_sequences.hard.q9;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    //Java
+
+    String[] jurisdictions = new String[]{
+            "Alberta",
+            "British Columbia",
+            "Manitoba",
+            "New Brunswick",
+            "Newfoundland and Labrador",
+            "Northwest Territories",
+            "Nova Scotia",
+            "Nunavut",
+            "Ontario",
+            "Prince Edward Island",
+            "Quebec",
+            "Saskatchewan",
+            "Yukon"
+    };
+
+    @Override
+    public Clause[] testSentence() {
```

Comment:
The problem with this PR is that you didn't declare it was an input test, that is why it is failing

Reply:
I made that change and it is still failing